aten/src/ATen/core/library.cpp

```diff
@@ -38,7 +38,9 @@ namespace {
     }
     return "(unknown)";
   }
-}
+
+  constexpr auto CatchAll = c10::DispatchKey::CatchAll;
```

Comment:
Why do you need this? You can't just use the original as-is?

Reply:
Just for shortness (i.e. `constexpr auto` is just a syntactic sugar saying I don't want to type `c10::Dispatch::Catchall` all the time :) )